WooCommerce/WooCommerceTests/POS/ViewModels/PointOfSaleCardPresentPaymentOnboardingViewModelTests.swift

```diff
@@ -0,0 +1,33 @@
+import XCTest
+@testable import WooCommerce
+
+final class PointOfSaleCardPresentPaymentOnboardingViewModelTests: XCTestCase {
+    func test_onDismissTap_is_invoked_when_cancelOnboarding_is_called() throws {
+        // Given
+        var isDismissTapInvoked = false
+        let sut = PointOfSaleCardPresentPaymentOnboardingViewModel(onboardingViewModel: .init(fixedState: .genericError),
+            onDismissTap: {
+                isDismissTapInvoked = true
+            })
+
+        // When
+        sut.cancelOnboarding()
+
+        // Then
+        XCTAssertTrue(isDismissTapInvoked)
+    }
+
+    func test_onboardingURL_is_set_when_onboarding_vm_showURL_is_invoked() throws {
+        // Given
+        var isDismissTapInvoked = false
```

Comment:
I think we don't need this one:
```suggestion
```

Reply:
Nice catch, removed in https://github.com/woocommerce/woocommerce-ios/pull/14224/commits/4ab6e1aad87a93492315e0bc8e8ac8c6923a5b65.